src/FlexForms/Rector/RenderTypeFlexFormRector.php

```diff
@@ -18,7 +18,8 @@ final class RenderTypeFlexFormRector implements FlexFormRectorInterface
     public function transform(DOMDocument $domDocument): bool
     {
         $xpath = new DOMXPath($domDocument);
-        /** @var DOMNodeList|DOMElement[] $elements */
+
+        /** @var DOMNodeList<DOMElement> $elements */
```

Comment:
This is PHPStan syntax for what was here before. 

Basically "`DOMNodeList` is array collection object of `DOMElement` object"